src/test/java/core/basesyntax/service/impl/ReportServiceImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportService;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReportServiceImplTest {
+    private static final String REPORT = "fruit,quantity" + System.lineSeparator()
+            + "banana,152" + System.lineSeparator() + "apple,90";
+    private static final String EMPTY_REPORT = "fruit,quantity" + System.lineSeparator();
+    private ReportService reportService;
+
+    @Before
+    public void setUp() {
+        reportService = new ReportServiceImpl();
+    }
+
+    @Test
+    public void createReport_Ok() {
```

Comment:
pay attention to your method namings, and let's make them consistent: 
<methodUnderTest>_<state>_<expectedBehavior>